build/codegen/codegen.sh

```diff
@@ -62,6 +62,4 @@ find "${scriptdir}/../../pkg/client" -name "*.go" -exec \
 find "${scriptdir}/../../pkg/client" -name "*.go" -exec \
     $SED 's/cephNFSs/cephNFSes/g' {} +
 find "${scriptdir}/../../pkg/client" -name "*.go.bak" -delete
-find "${scriptdir}/../../pkg/apis" -name "zz_generated.deepcopy.go" -exec \
```

Comment:
Why is this gone?

Reply:
We added this earlier so that gosec linter doesn't check these autogenerated files and golangci linter by default not reading these autogenerated files I believe it looks for `Code generated by x` pattern and then linter doesn't check those file.